src/fhir/1_Questionnaire-C-IDAS.json

```diff
@@ -26,7 +26,7 @@
         "extension": [
            {
              "url": "http://hl7.org/fhir/StructureDefinition/rendering-xhtml",
-             "valueString": "<h4>Anxiety</h4><div>Anxious expression, ruminations, worrying</div>"
+             "valueString": "<h4>Anxiety</h4><div>In the last week have you observed <b>Anxiety</b> in your care recipient? (anxious expression, ruminations, worrying)</div>"
```

Comment:
@sierramatice, same here, the `text` field for this question should also be updated.